wgsl/index.bs

```diff
@@ -7755,18 +7775,15 @@ that value is ignored.
 
 ## Const Assertion Statement ## {#const-assert-statement}
```

Comment:
I expected this to move into the assertions section.

Reply:
@alan-baker thank you for the review. I tried to follow the suggestion and updated the preview (and pushed the commit). Please let me know if this update misses anything. Appreciated!